packages/integration-tests/tests/api/tokens.test.ts

```diff
@@ -113,6 +113,26 @@ describe("Tokens", () => {
         });
       });
 
+      //id1907
+      it("Verify /api?module=token&action=tokeninfo response", async () => {
+        await helper.retryTestAction(async () => {
+          apiRoute = `/api?module=token&action=tokeninfo&contractaddress=${Token.ETHER_ERC20_Address}`;
+          response = await helper.performGETrequest(apiRoute);
```

Comment:
It's make sense to align it with all functions and send correct network instead.